docker/build

```diff
@@ -3,6 +3,7 @@ set -xeu
 
 # Install dependencies
 apt-get install -y \
+    d1to2fix=0.10.3* \
```

Comment:
Oh, I don't think 0.10.x is present in any cachalot image, does it have important changes to justify a cachalot release? Is it backwards incompatible or can it be done as a "patch" release of cachalot?

Also, why the `.3*`, that would match `0.10.3` and `0.10.34` but not `0.10.4`, which sounds really weird.